### DISCUSSION TITLE
Pandas-to-table wrapper

### DISCUSSION THREAD

**Discussion by bryanwhiting** [h2oai/wave#1787] [Ideas]
it'd be helpful to have a cleaner interface for converting pandas tables into `ui.table`. R has `reactable` and `DT` packages that make it really easy to include interactive tables.

Probably a poor design, but something like this where you could just pass a simple dictionary into the table:
```
from h2o_wave import ui
from h2o_wave.table import col

ui.table(
pandas_df = df,
name='issues',
col_defs = {"text": col(label='Text'), "dollar": col(label="Dollar (USD)", format = "dollar")}
groupable=True,
downloadable=True,
resettable=True,
height='600px'
)
```
(Adapted from https://wave.h2o.ai/docs/examples/table). This design would assume a default style.  See https://glin.github.io/reactable/articles/examples.html#column-definitions for example.

**Comment by mturoci** [comment]
Yes, this is something that we have been talking about a few times and should definitely be added (maybe as a separate package). The only problem is lack of time currently.

At the very minimum, we would like to make it easy to convert dataframes to Wave tables/plots out of the box.
